app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -242,16 +262,25 @@ export default function ManageChannelMembers({
             if (users.length >= PER_PAGE_DEFAULT) {
                 hasMoreProfiles.current = true;
             }
+            if (users.length < PER_PAGE_DEFAULT) {
+                hasMoreProfiles.current = false;
+            }
```

Comment:
I guess this makes more sense to have it set to `true` at mount, and just set it to false here. But not 100% sure. What do you think?

Reply:
I think it makes more sense as well, there's no need to set it to `true` on each request.

Reply:
I think it makes more sense as well, there's no need to set it to `true` on each request.